### PR TITLE
Update troubleshooting.asciidoc

### PR DESCRIPTION
**Pull request by EricDavisX**
This can help resolve questions around install of Fleet Sever component as part of Elastic Agent usage, and adds several helpful tips in about:
- port usage
- --insecure
- ca.crt full path is needed

Feedback is coming in from 7.13 difficulties and it would be nice to give uses all the help we can.

### CODE REVIEW

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 1]
```diff
@@ -308,6 +309,34 @@ agent you want to unenroll.
...
+
+This error message can be shown for different reasons but a common one, when attempting production-like usage,
+is seen when the ca.crt file passed in cannot be found.  One way to verify if this is the problem you are
+seeing is to bootstrap {fleet-server} without passing a ca.crt file, which implies you would test any subsequent
+{agent} installs temporarily with {fleet-sever}'s own self-signed cert.
+
+Tip: be sure to pass in the full path to the ca.crt file, a relative path is not viable.
+
+Further, you will know your {fleet-server} is set up with its testing oriented self-signed certificate usage
+if you see the following error during {agent} installs:
```
Comment (EamonnTP) [inline docs/en/ingest-management/troubleshooting.asciidoc:324]:
```suggestion
You will know if your {fleet-server} is set up with its testing oriented self-signed certificate usage,
when you see the following error during {agent} installs:
```

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 2]
```diff
@@ -308,6 +309,34 @@ agent you want to unenroll.
...
+
+[source,sh]
+----
+Error: fail to enroll: fail to execute request to fleet-server: x509: certificate signed by unknown authority
+Error: enroll command failed with exit code: 1
+----
+
+To install or enroll against a self-signed cert {fleet-server} {agent} add in the '--insecure' option to the
+command as below:
```
Comment (EamonnTP) [inline docs/en/ingest-management/troubleshooting.asciidoc:333]:
```suggestion
To install or enroll against a self-signed cert {fleet-server} {agent}, add in the `--insecure` option to the
command:
```

**Review by EamonnTP — APPROVED** [review]
Looks good. Just a couple of suggestions.

**Comment by EricDavisX** [comment]
Those all look great, thanks for prepping them for me, @EamonnTP !

**Pull request merged**